features/simple.feature

```diff
@@ -29,4 +29,24 @@ Feature: User simple flows - sign in and registeration
       | addressPostCode | ABC 123    |
       | dateOfBirth     | 1987-03-03 |
     When they give their consent
-    Then they should be successfully verified
\ No newline at end of file
+    Then they should be successfully verified
+
+    Scenario: User registers with documents
```

Comment:
This has pretty much the same steps as that of "Scenario: User registers, confirms identity and signs in" in "non_repudiation.feature". It could probably be excluded, I think.
The "non_repudiation.feature" test has a few extra steps but it covers the Scenario you added.
Perhaps the non_repudiation.feature could be changed to say "Feature: User registers with documents, returns to confirm identity and signs in successfully"

Reply:
Good spot. Will make that change.